modules/vba_documents/db/migrate/20180411001427_create_vba_documents_upload_submissions.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
```

Comment:
Even though the goal is for these to be self contained, these affect global application state. How would you feel about pushing migrations into the root of vets-api?

Reply:
The tables get namespaced to alleviate that somewhat, but agreed it's kind of funky how top-level db/schema.rb will not reflect top-level db/migrate folder.

I think there's an engines workflow for copying the migrations up to top-level, let me look into that.